airbyte-ci/connectors/pipelines/pipelines/airbyte_ci/connectors/test/steps/common.py

```diff
@@ -443,14 +443,13 @@ async def _run(self, current_acceptance_tests_result: StepResult) -> StepResult:
             )
 
 
-class RegressionTests(Step):
-    """A step to run regression tests for a connector."""
+class AbstractLiveTests(Step):
+    """A step to run live tests for a connector."""
 
     context: ConnectorContext
-    title = "Regression tests"
     skipped_exit_code = 5
     accept_extra_params = True
-    regression_tests_artifacts_dir = Path("/tmp/regression_tests_artifacts")
+    tests_artifacts_dir = Path("/tmp/live_tests_artifacts")
```

Comment:
nit: I'm not sure if this path refers to a local one or an in-container one. A comment or a rename might help.